documentation/running_microservice.md

```diff
@@ -0,0 +1,90 @@
+# Running the microservice
+
+### Docker Support
+The Student Support Service provides a `Dockerfile` for building a Docker image and a `docker-compose.yml` for running a Docker container that hosts the service `.jar`.  For the following command to work, the Docker Engine must be installed on the target build machine.  Resources for downloading and installing the Docker Engine on various operating systems can be found [here](https://docs.docker.com/engine/installation/).  For details on what Docker is and how it works, refer to [this page](https://www.docker.com/what-docker).
+
+To build the service and its associated Docker image:
+
+* `mvn clean install docker:build -f /path/to/service/pom.xml`
+
+## Run
+### Run in IDE
+To run the Student Support Service in the IDE, update the `spring.datasource.url`, `spring.datasource.username`, `spring.datasource.password` properties defined in the `service/src/main/resources/application.properties` file and set them to appropriate values.
+
+### Run .JAR
+To run the compiled jar built by one of the build commands above, use the following:
+
+```
+java -Xms256m -Xmx512m \
+    -jar /path/to/target/tds-student-service-0.0.1-SNAPSHOT.jar \
```

Comment:
Minor point; wouldn't the version number change to whatever's specified in the `pom.xml`?  I think maybe something like `/path/to/target/tds-student-service-[version in pom.xml].jar`.  Or maybe call out that the version number should be specified in the command.